### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by etaletai13**
remove extra step from build by installing nw and nwjs-builder as a devDependency - snyk also is moved to devDeps. No global installs on the user's machine needed and no extra packages in the final product.

### CODE REVIEW

**Comment by cryptofuture** [comment]
Does snyk patching works that way?
Also we don't need, since nw comes from binary package and we use another builder.
```
"nw": "^0.29.3",
"nwjs-builder": "^1.14.0",
```

**Comment by etaletai13** [comment]
@cryptofuture Try taking those out and testing the package. Test the way snyk is set up; and test the way it currently runs. None of what you just said is accurate.

**Comment by etaletai13** [comment]
@cryptofuture Also, another builder? Because the build instructions use nwjs-builder. If it's being built differently on the production end, one of the two is Wrong, and you need to address that.

**Comment by cryptofuture** [comment]
@etaletai13 Basically to develop you don't need builder at all, all you need is sdk version ow nwjs.
That not denies the fact that you could build with nwjs-builder also.
But on builder we use web2exe:
https://github.com/cryptofuture/music-builder

**Comment by etaletai13** [comment]
@cryptofuture Why do you involve so much complexity?

**Comment by cryptofuture** [comment]
@etaletai13 Explain yourself.
Building is mostly the same using nwjs-builder and web2exe, but web2exe gives more control under result. Also we had some problems with nwjs-builder builds.

**Comment by etaletai13** [comment]
@cryptofuture Then fix the readme.

**Comment by Varunram** [comment]
The README shows only one way the app can be packages, not all of them. So it isn't really necessary to update it. As @cryptofuture said, both web2exe and nwjs-builder work in similar ways, just that we felt it is easier for people to use nwjs-builder to package the app since they don't need to configure a whole lot of things.

File: package.json [hunk 1]
```diff
@@ -33,11 +33,15 @@
...
-    "snyk": "^1.70.2",
     "web3": "^0.20.6",
     "y18n": "^4.0.0",
     "zxcvbn": "^4.4.2"
   },
+  "devDependencies": {
+    "nw": "^0.29.3",
+    "nwjs-builder": "^1.14.0",
+    "snyk": "^1.71.0"
+  }
```
Comment (Varunram) [inline package.json]: need a comma here, this is JSON

Comment (etaletai13) [reply package.json]: The weird part is ... why did that work on my machine!? That's not even valid.

**Comment by etaletai13** [comment]
Changes made. As for snyk - as a devDependency it functions identically; that's where it was built to live. It's not part of the actual app.

**Pull request merged**